### PR TITLE
Plugin py docs fixups

### PR DESCRIPTION
**Pull request by ritch**

### CODE REVIEW

File: fiftyone/operators/__init__.py [hunk 1]
```diff
@@ -4,6 +4,12 @@
 |
 """
 
+import types
+import wrapt
```
Comment (imanjra) [inline fiftyone/operators/__init__.py:8]: Is this used?

File: fiftyone/operators/__init__.py [hunk 2]
```diff
@@ -4,6 +4,12 @@
...
 from .operator import Operator, OperatorConfig
 from .registry import register_operator
 from .executor import execute_operator
+
+# This enables Sphinx refs to directly use paths imported here
+# and ignore functions marked as depreciated
+__all__ = [k for k, v in globals().items() if not k.startswith("_")]
```
Comment (ritch) [inline fiftyone/operators/__init__.py:15]: Not actually certain this is needed. Let me try again without it.

**Review by imanjra — APPROVED** [review]

File: fiftyone/operators/__init__.py (hunk 2 above)
Comment (ritch) [reply fiftyone/operators/__init__.py:15]: image

This is without.

Comment (ritch) [reply fiftyone/operators/__init__.py:15]: With:

image

File: fiftyone/operators/__init__.py (hunk 1 above)
Comment (ritch) [reply fiftyone/operators/__init__.py]: Removed
